contrib/examples/deploy-playbook-mock.yaml

```diff
@@ -0,0 +1,61 @@
+---
+kind: Deployment
+apiVersion: extensions/v1beta1
```

Comment:
I got a failure here now, should this be "apps/v1beta1"? Or am I running into 3.8 oc client problems.

Failure was:

```
TASK [deploy playbook-mock for testing with fake-openshift-ansible] *******************************************************************************************************************************************************************
Friday 05 January 2018  14:17:18 -0400 (0:00:00.422)       0:00:01.012 ********                                    
fatal: [localhost]: FAILED! => {"changed": true, "cmd": "oc create -n cluster-operator -f /home/dgoodwin/go/src/github.com/openshift/cluster-operator/contrib/ansible/../examples/deploy-playbook-mock.yaml", "delta": "0:00:00.167099", "end": "2018-01-05 14:17:19.236447", "msg": "non-zero return code", "rc": 1, "start": "2018-01-05 14:17:19.069348", "stderr": "Error from server (BadRequest): error when creating \"/home/dgoodwin/go/src/github.com/openshift/cluster-operator/contrib/ansible/../examples/deploy-playbook-mock.yaml\": Deployment in version \"v1beta1\" cannot be handled as a Deployment: [pos 354]: json: expect char '\"' but got char '1'", "stderr_lines": ["Error from server (BadRequest): error when creating \"/home/dgoodwin/go/src/github.com/openshift/cluster-operator/contrib/ansible/../examples/deploy-playbook-mock.yaml\": Deployment in version \"v1beta1\" cannot be handled as a Deployment: [pos 354]: json: expect char '\"' but got char '1'"], "stdout": "service \"playbook-mock\" created", "stdout_lines": ["service \"playbook-mock\" created"]}     
```

Reply:
Tried with 3.9 and oc apply but something still doesn't like this file.